google-chart.html

```diff
@@ -338,19 +338,43 @@
     // Promise for the DataView with which the chart is drawn.
     _dataView: null,
 
+    // Promise for the drawn chart.
+    _chartDrawn: null,
+
+    // The current selection in the chart.
+    _selection: null,
+
     // The chart type was changed.
     // We need to create a new chart and redraw.
     _typeChanged: function() {
-      this._chart = this._createChart();
+      this._chart = createChart(this.type, this.$.chartdiv)
+          .then(this._setupChart.bind(this));
```

Comment:
Right. That should be OK then. You should also really load the polyfill import in this component too. If iron-ajax is ever removed as the implementation, we'll still want it as a dependency.  And add it to bower.json.